js/coinsbit.js

```diff
@@ -0,0 +1,1070 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, ArgumentsRequired, InvalidOrder, OrderNotFound } = require ('./base/errors');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class coinsbit extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'coinsbit',
+            'name': 'Coinsbit',
+            'countries': [ 'EE' ], // Estonia
+            'rateLimit': 1000, // No defaults known
+            'version': 'v1',
+            'certified': false,
+            'hostname': 'coinsbit.io', // or coinsbit.in
+            'has': {
+                'createOrder': true,
+                'cancelOrder': true,
+                'cancelAllOrders': false,
+                'createMarketOrder': true,
+                'fetchBalance': true,
+                'fetchDepositAddress': false,
+                'fetchDeposits': false,
+                'fetchFundingRateHistory': false,
+                'fetchMarkets': true,
+                'fetchMyTrades': false,
+                'fetchOrderTrades': true,
+                'fetchOHLCV': true,
+                'fetchOrder': false,
+                'fetchOrders': false,
+                'fetchOpenOrder': false,
+                'fetchOpenOrders': true,
+                'fetchClosedOrder': false,
+                'fetchClosedOrders': true,
+                'fetchCanceledOrders': false,
+                'fetchOrderBook': true,
+                'fetchPosition': false,
+                'fetchPositions': false,
+                'fetchStatus': false,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': false,
+                'fetchTrades': true,
+                'fetchWithdrawals': false,
+                'deposit': false,
+                'withdraw': false,
+                'addMargin': false,
+                'reduceMargin': false,
+                'setLeverage': false,
+            },
+            'timeframes': {
+                '1m': 60, // works for max 12 hours diapason (between from-to)
+                '30m': 1800, // for max 24 hours diapason
+                '1h': 3600, // for max 7 days diapason
+                '1d': 86400, // for any diapason
+            },
+            'urls': {
+                'logo': '  <<<TODO>>>   https://upload.wikimedia.org/wikipedia/commons/8/8c/Coinsbit.png  ',
+                'api': {
+                    'public': 'https://{hostname}',
+                    'private': 'https://{hostname}',
+                },
+                'www': 'https://{hostname}/',
```

Comment:
We don't implode the API hostname into the Web URL – that remains pointing to coinsbit.io.